src/rgw/rgw_lua_utils.cc

```diff
@@ -24,6 +27,47 @@ int RGWDebugLog(lua_State* L)
   auto message = luaL_checkstring(L, 1);
   ldout(cct, 20) << "Lua INFO: " << message << dendl;
   return 0;
+
+}
+
+int RGWDebugLog2(lua_State* L)
+{
+  auto cct = reinterpret_cast<CephContext*>(lua_touserdata(L, lua_upvalueindex(FIRST_UPVAL)));
+
+  fmt::dynamic_format_arg_store<fmt::format_context> store;
+  std::string format = luaL_checkstring(L, 1);
+
+  int num_args = 0;
+  size_t pstart = 0, pend = 0;
+
+  // Does not handle escaping or positional arguments
+
+  pend = format.find("{}");
```

Comment:
what if someone uses: "{ }" ?
maybe better to detect opening and closing brackets separatly?